Makefile

```diff
@@ -104,12 +104,34 @@ rpm: $(RPMFILE)
 $(RPMFILE): $(SRPMFILE) bots
 	test/rpmbuild-local $(SRPMFILE)
 
+# pybridge scenario: build and install the python bridge from cockpit repo
+# this is a giant hack to test the pybridge in a scenario before it gets packaged into cockpit-bridge.rpm
+ifeq ("$(TEST_SCENARIO)","pybridge")
+COCKPIT_PYBRIDGE_REF = main
+
+tmp/pybridge:
+	# no pip on FCOS
+	rm -rf tmp/pybridge
+	git init tmp/pybridge
+	git -C tmp/pybridge remote add origin https://github.com/cockpit-project/cockpit
+	git -C tmp/pybridge fetch --depth=1 origin ${COCKPIT_PYBRIDGE_REF}
+	git -C tmp/pybridge reset --hard FETCH_HEAD
+	tmp/pybridge/tools/systemd_ctypes
+
+VM_DEPENDS = tmp/pybridge
+VM_CUSTOMIZE_FLAGS = --upload tmp/pybridge/modules/systemd_ctypes/src/systemd_ctypes:/tmp/pymods \
+	--upload tmp/pybridge/src/cockpit:/tmp/pymods \
+	--run-command 'P=$$(python3 -c "import sysconfig; print(sysconfig.get_path('"'"'purelib'"'"'))"); \
+	               mkdir -p $$P; cp -r /tmp/pymods/* $$P' \
+	--run-command 'echo "python3 -m cockpit.bridge \"\$$@\"" > /usr/local/bin/cockpit-bridge; chmod a+x /usr/local/bin/cockpit-bridge'
```

Comment:
Please use either pip or python3-installer for this.

Reply:
Specifically: we could modify the install command for wheels to check if /usr is writable and install to /usr/local otherwise.  Or just always /usr/local.

Reply:
I already had a local change to `image-customize` to install into /usr/local/ -- fully agreed that we wan this. This would be conceptually the right thing to do, plus it would allow us to more easily switch between C and py bridge in a prepared image (something that I often sorely miss). However, a lot of our tests hardcode/assume /usr/bin/cockpit-bridge, so we'll need some fixes first.

Neither pip nor -installer are part of fedora-coreos, and these will not be runtime dependencies of cockpit-bridge, so I don't want to do this. That's what I described in the description -- but that's fine for now. We can agree to disagree and leave this as a draft PR. The utility is to have a vehicle to run and investigate tests, not so much in actually landing this PR.

Reply:
https://github.com/cockpit-project/cockpit/pull/18301

Reply:
FTR: I find that waaaaaay to much overhead, and carrying around untested/half-tested dead weight in cockpit.git. I only want a draft PR as a vehicle to investigate the ostree failures, this is good enough. (Again: I don't need/want to land this PR)